client/landing/gutenboarding/onboarding-block/designs/index.tsx

```diff
@@ -30,29 +30,34 @@ const Designs: React.FunctionComponent = () => {
 	const { goBack, goNext } = useStepNavigation();
 
 	const { setSelectedDesign, setFonts, resetFonts } = useDispatch( ONBOARD_STORE );
-	const { getSelectedDesign, hasPaidDesign, getRandomizedDesigns } = useSelect( ( select ) =>
-		select( ONBOARD_STORE )
-	);
-	const isAnchorFmSignup = useIsAnchorFm();
 
-	const selectedDesign = getSelectedDesign();
+	const { getSelectedDesign, hasPaidDesign, getRandomizedDesigns } = useSelect( ( select ) => {
```

Comment:
`getSelectedDesign` and `getRandomizedDesigns` are named like they are some callable functions. Should we use `selectedDesign` and `randomizedDesigns` instead?